apps/web/src/views/GaugesVoting/components/Table/GaugesTable/index.tsx

```diff
@@ -60,7 +60,7 @@ export const GaugesTable: React.FC<
           <Skeleton height={64} />
         </AutoColumn>
       ) : (
-        <tbody>
+        <div>
```

Comment:
may I ask what's the current issue? From html perspective div under table will be throwing a warning?

Reply:
<img width="1201" alt="image" src="https://github.com/pancakeswap/pancake-frontend/assets/144641937/ac7878de-05c3-42d2-81c9-577120b1c47c">

Reply:
hmm alright, but let's fix the style issue afterwards. Either we stick to the table convention or we don't use table related tags at all